modules/custom/core/govcms8_default_content/govcms8_default_content.info.yml

```diff
@@ -0,0 +1,5 @@
+name: GovCMS 8 Default Content
+type: module
+description: Provides default content for G8BT.
```

Comment:
Would placing this module under modules/custom/optional be a better place for it (at the moment) to offer people the option to install it up front (alongside the content types)?We